headers/functions/overlay29.h

```diff
@@ -617,14 +617,14 @@ void ExecuteMoveEffect(undefined4* param_1, struct entity* attacker, struct move
 bool DoMoveDamageInlined(struct entity* attacker, struct entity* defender, struct move* move,
                          enum item_id item_id);
 int DealDamage(struct entity* attacker, struct entity* defender, struct move* move,
-               int damage_mult_fp, enum item_id item_id);
+               fx32_8 damage_mult, enum item_id item_id);
 int DealDamageWithTypeAndPowerBoost(struct entity* attacker, struct entity* defender,
-                                    struct move* move, int damage_mult_fp, enum item_id item_id,
+                                    struct move* move, fx32_8 damage_mult, enum item_id item_id,
                                     enum type_id attack_type, int16_t power_boost);
 int DealDamageProjectile(struct entity* attacker, struct entity* defender, struct move* move,
-                         int power, int damage_mult_fp, enum item_id item_id);
+                         int power, fx32_8 damage_mult, enum item_id item_id);
 int DealDamageWithType(struct entity* attacker, struct entity* defender, enum type_id attack_type,
-                       struct move* move, int damage_mult_fp, enum item_id item_id);
```

Comment:
Can we still keep all the `_fp` suffixes? It makes it easier to read decompiled code in Ghidra.

Reply:
I thought they would no longer be necessary now that the proper type is used for the parameter, but sure.